domains/gsts-dev-01.is-an.app.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "../schemas/domain.schema.json",
+    "description": "Backend for an online shopping app",
+    "domain": "is-an.app",
+    "subdomain": "gsts-dev-01",
+    "owner": {
+      "repo": "https://github.com/gateway-stream/gateway_be",
+      "email": "percival.rapha@gmail.com"
+    },
+    "record": {
+      "A": ["34.201.17.130"]
+    },
+    "proxy": false

```

Comment:
Could you let me know if you are sure about that? Proxying can help protect your backend

Reply:
What kind of limitations, is this the same as Cloudflare Proxying?

Reply:
This is it